chapters/functions.tex

```diff
@@ -2283,10 +2283,11 @@ \subsection{Return Type Mapping}\label{return-type-mapping}
 \lstinline!String!  & \lstinline[language=C]!const char*! & \emph{Not allowed}\\
 \lstinline!T[$\mathit{dim}_{1}$, $\ldots$, $\mathit{dim}_{n}$]! & \emph{Not allowed} & \emph{Not allowed} \\
 Enumeration type    & \lstinline[language=C]!int!         & \lstinline[language=FORTRAN77]!INTEGER!\\
-Record              & See \cref{records}                  & \emph{Not allowed}\\
+Record              & \lstinline[language=C]!struct CRec! & \emph{Not allowed}\\
 \hline
 \end{tabular}
 \end{center}
+For the contents of \lstinline[language=C]!struct CRec! see \cref{records}.
```

Comment:
The reference to `\cref{records}` could be moved somewhere there is no need to speak of a `CRec`:
```suggestion
```